### DISCUSSION TITLE
Blynk gravity in SG

### DISCUSSION THREAD

**Discussion by mr-bridger** [universam1/iSpindel#523] [Q&A]
Hi all, 
Just had to update my iSpindel from the old ubidots, and thought id set up Blynk. 

All went successfully, but id like my gravity to show in SG instead of plato/brix. is it possible?

thanks

**Comment by thegreatgunbantoad** [comment]
It's all to do with how you generate your polynomial.  The standard calculators here are generally Sg or Plato, but you can use anything you like.  Simplest way is grab a load of tilt readings and your equivalent SG/brix/Plato at that tilt, feed it into something like Excel and have it generate a graph and an equation for you.  I think either:
https://github.com/universam1/iSpindel/blob/master/docs/Kalibrierung_en.xlsm
or
https://github.com/universam1/iSpindel/blob/master/docs/ManualPolyGenerator.xlsx
will work if you just use plato/brix in the relevant column.

**Reply by mr-bridger** [reply]
Ah, ok i used the Kalibrierung_en.xlsm to calculate my formula originally (Plato), but i had the conversion in ubidots to correct it to SG. there's no way of doing that in Blynk? the iSpindel is in my beer at the moment ;-)

**Comment by ErikdBr** [comment]
Or you can use this one: https://www.ispindel.de/tools/calibration/calibration.htm
There you can choose if you want the formula in SG or Plato.

**Reply by mr-bridger** [reply]
Looks like ill have to wait until my beer finishes and start calibration again, as its in the beer at the moment :)

**Reply by ErikdBr** [reply]
Can't you make a widget in Blynk with a formula that converts Plato to SG ?

**Reply by mr-bridger** [reply]
Thats what i hoped, but from all my searching i cannot find a way to do it.

**Comment by m0n5t3r** [comment]
you could take the calibration dataset and apply the formula to convert the gravity to SG -> the formula will give you SG

a quick search yielded this http://betatestbrewing.com/pages/plato_to_sg.html (which is pretty convenient because it lists the formula - when writing my own scripts I fished the calculations out of http://www.musther.net/vinocalc.html)
